.travis.yml

```diff
@@ -3,20 +3,29 @@ osx_image: xcode10
 env:
   global:
     - PROJECT=Bow.xcodeproj
+    - JOB="XCODE"
+    - DEST="OS=12.0,name=iPhone 8 Plus"
+    - SDK="iphonesimulator"
+    - ACTION="test"
   secure: LatJH8j809/6dhEO6WhpQyU3lAjtkOYHAqWD14+bU6lfAc5oRrRFO4B7h56gnIDu6tvYJS0Qa/IhIuB3zZQ/GlG12oivgE/FKp1xXxxHwqjmhPHdetX9YkeAbicJ8YrSQ+C0UC6NMmP1WtU7cmLBV4Q1VwdPnVvRIrOVuWdh3ITUzUz/qQMTZwi+6Z1w92QvxbyytXDqti62NrUq81ba2c7pvBbFBujMp5hyd6+EcuZojYWGNN3cIR2DcPVXD9j4kwLyDPVAcN/OROAwaJlOZ63FbsoUcZULOAX/6s5bmcU1CuYc+eBrXX7KifSqj3yUW00Ea0ynyvsn2GWldAw5b8cYgiLuqQBp3jxtPVH/tEWRdO5TdFtMl8CDEJQiblINjFCXRbnAh8fnk/9jxKTkK+e1DXNQEk/w61+vUB21j74Zw8PhPRvCsoV8XgPkd34UiSlG5XkPaj+vImRpB2/kemYxynpY2Sjq+qoK4iUJy/DXhX/agThWCpjY1hWL2LNL/24mk6DbUHZGVH+4RgoBJxBFCI7ucSp14Og0cCFmhFfsq0nttN+xNegosmHnEcv1Z4tRW750rPf2dgGlXkf6s4msbZ/kyRnccs2zlh3JYYac509Fk98FRNfLb9q0C/jLb0xpg6qf9WbI1QbMcxPHKakyGEUuo+7FO+CLzrJ0qns=
 matrix:
   include:
-    - env: JOB="XCODE" DEST="OS=12.0,name=iPhone 8 Plus" SCHEME="Bow" SDK="iphonesimulator" ACTION="test"
-    - env: JOB="XCODE" DEST="arch=x86_64" SCHEME="Bow-macOS" SDK="macosx" ACTION="build"
-    - env: JOB="XCODE" DEST="OS=12.0,name=Apple TV" SCHEME="Bow-tvOS" SDK="appletvsimulator" ACTION="build"
-    - env: JOB="XCODE" DEST="OS=5.0,name=Apple Watch Series 4 - 44mm" SCHEME="Bow-watchOS" SDK="watchsimulator" ACTION="build"
+    - env: SCHEME="Bow"
+    - env: SCHEME="Bow-Free"
+    - env: SCHEME="Bow-Generic"
+    - env: SCHEME="Bow-Optics"
+    - env: SCHEME="Bow-RecursionSchemes"
+    - env: SCHEME="Bow-Result"
+    - env: SCHEME="Bow-Effects"
+    - env: SCHEME="Bow-Rx"
```

Comment:
Integrations are named like `arrow-effects-rx`. Is there a reason to avoid the effects prefix here?

Reply:
These are the build scheme names, not the actual module names.

Reply:
Ok, I thought you had named them like BowRx and so on, as it's stated in the description.